config/default.yaml

```diff
@@ -34,7 +34,7 @@ storage:
   download_cache_path: client_storage_download_cache
   concurrent_upload_limit: 10
   concurrent_validation_limit: 10
-  upload_loop_interval: 500
+  upload_loop_interval: 2000
   request_timeout: 15000
```

Comment:
I would increase this one, since today I've seen some errors because of this. 30000 would be good (request_timeout)

Reply:
done.